src/backend/joanie/payment/backends/lyra/__init__.py

```diff
@@ -1,3 +1,5 @@
+# ruff: noqa: PLR0913
```

Comment:
you create a card token. You don't capture it. Otherwise the day we want to use the `capture` feature, the name will already be used.